modules/ocf_desktop/files/suspend/fix-daisy

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# when using daisy chaining on nvidia graphics card on debian,
+# primary monitor is on but the rest of the daisy chained
+# monitors are asleep, and does not wake up.
+#
+# below code forces the monitors to wake up
+
+xset dpms force off
+xset dpms force on
+xset dpms 0 0 0
```

Comment:
I think this line is disabling all monitors from going to sleep. We shouldn't be doing this. I thought forcing off and then back on was sufficient to fix the problem?

Reply:
Following up from in-person conversation:
- There was a reason the line `xset dpms 0 0 0` was necessary - mainly there was a case where the above two lines fixes it, but as soon as the displays go to sleep the same issue happnes again because the `fix-daisy` script does not get applied.
- Only apply this `fix-daisy` script to multiple monitor setup - since the vast majority of the desktops are single monitor setup, we won't be wasting as much energy than to disable the sleep everywhere.